keyboards/handwired/zergo/keymaps/default/keymap.c

```diff
@@ -16,22 +16,35 @@
 
 #include QMK_KEYBOARD_H
 
+// Each layer gets a name for readability.
+#define l_main 0
+#define l_arrows 1
```

Comment:
```suggestion
enum zergo_layers {
    l_main,
    l_arrows,
};
```

I personally think the `enum` is better here, but IMO using `#define`s is not a dealbreaker for only two layers.  Up to you.

Reply:
well, I'm not too deep in coding, so if that's preferable option I will add.